src/07-projects/projects.apib

```diff
@@ -230,13 +230,13 @@ Update a participant's role for a project.
 
 ### projects.updateInternalHourlyCost [POST /projects.updateInternalHourlyCost]
 
-Updates the internal hourly cost for a participant for one project, or all projects.
+Updates the internal hourly cost for a participant for one project.
 This does not update the global internal rate of the employee. See `users.updateInternalHourlyCost`.
 
 + Request (application/json;charset=utf-8)
 
     + Attributes (object)
-        + id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional) - If there is no specific project id, update for all projects
+        + id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, required)
```

Comment:
I don't know if you have to specify "required". Leaving out "optional" is enough.

Reply:
PP : I like the required to be present, it's more explicit